packages/maxima/build.sh

```diff
@@ -47,7 +52,7 @@ termux_step_host_build() {
 
 termux_step_make() {
 	local _PREFIX_FOR_BUILD=$TERMUX_PKG_HOSTBUILD_DIR/prefix
-	
+
```

Comment:
It looks like your formatter is taking out some tabs on occasion.
In this case, that's good though.
I'm not sure how this ever passed the trailing whitespace check.